packages/cadl-ranch-specs/http/server/versions/all-operations/main.tsp

```diff
@@ -0,0 +1,37 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@typespec/versioning";
+
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+@doc("Illustrates server with all operations defined with api-versions.")
```

Comment:
Can we start using doc comment instead of `@doc` 
See this PR for example https://github.com/Azure/cadl-ranch/pull/429/files#diff-0c3650d9ce1977c4f60dc4f5231c2b6be51d87d1c28b15548d74623ce89581e7